software/cloud-dashboard/react-client/src/components/temp-view/temp-plot.js

```diff
@@ -38,6 +40,22 @@ class TempPlot extends Component {
 		return array;
 	};
 	
+  filterDataMinMax = (array) => {
```

Comment:
This seems reasonable, but for the implementation of this function, consider using the `map` function as described [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) which will allow you to apply a function _f_ to each element _e_ in an array with the output _f(e)_ populating the corresponding index in an output array.

Reply:
Yeah, I wanted to use map... But I wasn't in Python. Syntax.  Bah.